LondonBritishLibrary/orient/BLorient580.xml

```diff
@@ -54,49 +96,69 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                                </extent>
-                                
-                                
+                                  </extent>
+<foliation>Two unnumbered blank leaves at the beginning.</foliation>
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
-                                <layout columns="2" writtenLines="16 19" cert="high"/>
+                                <layout columns="1" writtenLines="16 19" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is written in elegant hand of the  <date>19th century</date>
+                                <desc>The manuscript is written in elegant hand of the <date>19th century</date>
                                 </desc>
-                                
+
                                 <date notAfter="1800" notAfter-custom="1900"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
+
                         <additions>
-                            
+<list>
+<item xml:id="a1">
+<locus target="#23r"/>
+<desc type="Supplication"/>
+<q xml:lang="gez">ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ አድኅነኒ፡ እመከራ፡ ሥጋ፡ ወነፍስ፡ ለገብርከ፡ ገብረ፡ ማርያም፡ ወም<supplied reason="omitted">ስለ</supplied>፡ ብእሲትየ፡ ወውሉድየ፡
+ወወለትየ፡ ወለተ፡ እግዚአብሔር፡ አምኃ፡ ጊዮርጊስ፡ ይኩኖ፡ አምላክ፡ ወለተ፡ ኪዳን፡ ወልደ፡ <add place="above">ርቆሬዎስ፡</add></q>
+<note>According to the supplication the manuscript belonged to <persName role="owner" ref="PRS13539GabraMariam">Gabra Māryam</persName>. Mentioned are also his wife <persName ref="PRS13540Walatta">Walatta
+ʾƎgziʾabǝḥer</persName> and their children: <persName ref="PRS13541Amha">ʾAmḫā Giyorgis</persName>, <persName ref="PRS13542Yekunno">Yǝkunno ʾAmlāk</persName>, <persName ref="PRS13543WalattaK">Walatta Kidān</persName> and <persName ref="PRS13544WaldaM">Walda Marqorewos</persName>.</note>
+</item>
+
+</list>
                         </additions>
-                        
+
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                          <decoNote xml:id="b1">In the original boards.</decoNote>
+                                <decoNote xml:id="b2" type="bindingMaterial">
+                                    <material key="wood"/>
+                                  </decoNote>
+                            </binding>
+                        </bindingDesc>
                     </physDesc>
-                    
+
                     <history>
                         <origin>
-                            
+
                             <origDate/>
```

Comment:
please, add the date in accordance with palaegraphic features

Reply:
Done